index.js

```diff
@@ -12,8 +12,9 @@ if (process.env.APPLICATIONINSIGHTS_CONNECTION_STRING) {
     .setAutoCollectExceptions(true)
     .setAutoCollectDependencies(true)
     .setAutoDependencyCorrelation(true)
-    .setUseDiskRetryCaching(true)
-    .start();
+    .setUseDiskRetryCaching(true);
+  appInsights.defaultClient.config.samplingPercentage = process.env.APPLICATIONINSIGHTS_SAMPLING_PERCENTAGE ? process.env.APPLICATIONINSIGHTS_SAMPLING_PERCENTAGE : 100;
+  appInsights.start();
```

Comment:
Think this is a beautiful spot to use this:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing

Reply:
Yes, would have been nice, however, I forgot I have to use `parseInt()` on the value... This doesn't work with the 'nullish coalescing' method.